### PR TITLE
Add general details of the hike up to the cabin

### PR DESCRIPTION
**Pull request by pnicknish2**
I added a sentence in the "Location and Access" section to explain that there is a moderate hike with a steep hill involved to get up to the cabin.

### CODE REVIEW

File: _pages/rentals/intervale-manual.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ Intervale is located a few miles north of North Conway, NH. The exact location o
 
 Intervale is not road-accessible. Therefore, we have an easement which gives us a Right of Way (RoW) to access our property. Our RoW passes by a pair of other homes and directly adjacent the property of a third. It is important that you follow all of the rules below regarding our RoW when you access our property. We must respect our neighbors at all times. Failure to follow these procedures constitutes trespassing.
 
+Please note that accessing the cabin involves a moderate 250 meter hike, with uneven terrain and one rather steep hill. In winter, it can be useful to have spikes or some additional traction to assist with footing.
```
Comment (fpagnoux) [inline _pages/rentals/intervale-manual.md:40]: 250m feels like a low estimate, even without counting the walk around the road. Maybe we should mention that it's usually a 10 to 15 minutes hike from the parking lot?

**Comment by pnicknish2** [comment]
Yeah I guess 250m seems kind of low, but Josh mentioned it could be good to put a distance in. I am fine to have the time in there instead.

Paul

**Comment by JoshBradshaw** [comment]
I guess its more like 750m. I wanted to use a distance and description as opposed to time, because it's objective. I had a handicapped user reaching out to ask how accessible the cabin is, which prompted this change.

File: _pages/rentals/intervale-manual.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fpagnoux) [reply _pages/rentals/intervale-manual.md:40]: @pnicknish2 let me know if the suggestion above looks good to you ^

[the remark this answers is not in the document]
Comment (pnicknish2) [reply _pages/rentals/intervale-manual.md:40]: Yeah that looks good to me! Thanks!

**Pull request merged**